DOCKER/Dockerfile

```diff
@@ -38,8 +38,8 @@ RUN mkdir -p /deploy/pge/extractors/filename
 COPY chunkfile_extractor.xml /deploy/pge/extractors/filename/chunkfile_extractor.xml
 COPY chunkfile_metout.xml /deploy/pge/extractors/metout/chunkfile_metout.xml
 
-RUN cp /deploy/filemgr/lib/cas-filemgr-*.jar /deploy/workflow/lib/ 
-RUN cp /deploy/filemgr/lib/cas-filemgr-*.jar /deploy/resmgr/lib/ 
+RUN cp /deploy/filemgr/lib/cas-filemgr-*.jar /deploy/workflow/lib/
```

Comment:
Just my editor removing trailing whitespace by default, let me know if you want these lines reverted.